.codacy.yml

```diff
@@ -0,0 +1,18 @@
+engines:
+ rubocop:
+   enabled: true
+   exclude_paths:
+     - config/engines.yml
```

Comment:
we don't have such file in this repo

Reply:
My bad, I just copy pasted from my repo 😅